psdk_wrapper/src/psdk_wrapper.cpp

```diff
@@ -787,7 +787,7 @@ PSDKWrapper::initialize_ros_elements()
       create_publisher<geometry_msgs::msg::AccelStamped>(
           "psdk_ros2/acceleration_body_raw", 10);
   main_camera_stream_pub_ = create_publisher<sensor_msgs::msg::Image>(
-      "psdk_ros2/main_camera_stream", 10);
+      "psdk_ros2/main_camera_stream", rclcpp::SensorDataQoS());
   fpv_camera_stream_pub_ = create_publisher<sensor_msgs::msg::Image>(
       "psdk_ros2/fpv_camera_stream", 10);
```

Comment:
mising SensorDataQoS() for this one

Reply:
Added in https://github.com/umdlife/psdk_ros2/pull/35/commits/852779787fd05219c4087b17c92f449ad68fce37